From f1374be4173f290b4dcb82a01aad16ab86da46bb Mon Sep 17 00:00:00 2001
From: Christophe Priouzeau <christophe.priouzeau@foss.st.com>
Date: Wed, 13 Sep 2023 14:31:01 +0200
Subject: [PATCH 3/3] conf: add card config for stm32mp25 ev1 eval board

Add ALSA card configuration file for STM32MP25 EV1 board.

Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
---
 src/conf/cards/Makefile.am       |  3 ++-
 src/conf/cards/STM32MP25EV1.conf | 16 ++++++++++++++++
 src/conf/cards/aliases.conf      |  1 +
 3 files changed, 19 insertions(+), 1 deletion(-)
 create mode 100644 src/conf/cards/STM32MP25EV1.conf

diff --git a/src/conf/cards/Makefile.am b/src/conf/cards/Makefile.am
index 2ed47ad..6812cdf 100644
--- a/src/conf/cards/Makefile.am
+++ b/src/conf/cards/Makefile.am
@@ -61,7 +61,8 @@ cfg_files = aliases.conf \
 	VXPocket440.conf \
 	STM32MP15EV.conf \
 	STM32MP15DK.conf \
-	STM32MP13EV1.conf
+	STM32MP13EV1.conf \
+	STM32MP25EV1.conf
 
 if BUILD_ALISP
 cfg_files += aliases.alisp
diff --git a/src/conf/cards/STM32MP25EV1.conf b/src/conf/cards/STM32MP25EV1.conf
new file mode 100644
index 0000000..3cd5baf
--- /dev/null
+++ b/src/conf/cards/STM32MP25EV1.conf
@@ -0,0 +1,16 @@
+<confdir:pcm/hdmi.conf>
+
+STM32MP25EV1.pcm.hdmi.0 {
+        @args [ CARD AES0 AES1 AES2 AES3 ]
+        @args.CARD { type string }
+        @args.AES0 { type integer }
+        @args.AES1 { type integer }
+        @args.AES2 { type integer }
+        @args.AES3 { type integer }
+        type asym
+        playback.pcm {
+                type hw
+                card $CARD
+                device 0
+        }
+}
diff --git a/src/conf/cards/aliases.conf b/src/conf/cards/aliases.conf
index d1897c9..14f4f59 100644
--- a/src/conf/cards/aliases.conf
+++ b/src/conf/cards/aliases.conf
@@ -60,6 +60,7 @@ VC4-HDMI cards.vc4-hdmi
 STM32MP15-EV cards.STM32MP15EV
 STM32MP15-DK cards.STM32MP15DK
 STM32MP13-EV1 cards.STM32MP13EV1
+STM32MP25-EV1 cards.STM32MP25EV1
 
 <confdir:ctl/default.conf>
 <confdir:pcm/default.conf>
-- 
2.34.1

